pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -96,34 +96,31 @@
     "PadStorageInitialWaterQuality",
 ]
 
-# user needs to provide the path to the case study data file
-# for example: 'C:\\user\\Documents\\myfile.xlsx'
-# note the double backslashes '\\' in that path reference
-with resources.path(
-    "pareto.case_studies",
-    # "input_data_generic_strategic_case_study_LAYFLAT_FULL.xlsx",
+### Input Settings
+""" Please enter the path to folder in which the different input files reside
+    for example: 'C:\\user\\Documents\\myfile.xlsx'
+    note the double backslashes '\\' in that path reference
+    input_files should contain all files to be read. Each file will be a 
+    different scenario and create a separate output file"""
+input_folder = "pareto.case_studies"
+input_files = [
+    "input_data_generic_strategic_case_study_LAYFLAT_FULL.xlsx",
     "small_strategic_case_study.xlsx",
-    # "strategic_water_treatment_toy_case_study_t10.xlsx"
-) as fpath:
-    [df_sets, df_parameters] = get_data(fpath, set_list, parameter_list)
+]
+
 
-# create mathematical model
+### Optimization Settings
 """Valid values of config arguments for the default parameter in the create_model() call
- objective: [Objectives.cost, Objectives.reuse]
- pipeline_cost: [PipelineCost.distance_based, PipelineCost.capacity_based]
- pipeline_capacity: [PipelineCapacity.input, PipelineCapacity.calculated]
- node_capacity: [IncludeNodeCapacity.True, IncludeNodeCapacity.False]"""
-strategic_model = create_model(
-    df_sets,
-    df_parameters,
-    default={
-        "objective": Objectives.cost,
-        "pipeline_cost": PipelineCost.distance_based,
-        "pipeline_capacity": PipelineCapacity.input,
-        "node_capacity": IncludeNodeCapacity.true,
-    },
-)
+    objective: [Objectives.cost, Objectives.reuse]
+    pipeline_cost: [PipelineCost.distance_based, PipelineCost.capacity_based]
+    pipeline_capacity: [PipelineCapacity.input, PipelineCapacity.calculated]
+    node_capacity: [IncludeNodeCapacity.True, IncludeNodeCapacity.False]"""
+objective = Objectives.cost
```

Comment:
You can create the default object over here and pass it as an argument to the create_model function

Reply:
Done.